test/test-remote/test_alerts.ts

```diff
@@ -467,7 +467,7 @@ suite("End-to-end alert tests", function () {
     // See also: https://github.com/opstrace/opstrace/issues/1130
     if (deletedDefault || deletedSystem) {
       log.info("Waiting 15s after configs were deleted");
-      sleep(15.0);
+      await sleep(15.0);
```

Comment:
I still want a linter to at least emit a warning on these -- wow. This happens so so so often. https://github.com/eslint/eslint/issues/13567